packages/sdk/src/swap/sdk.ts

```diff
@@ -78,19 +74,16 @@ const assertQuoteValid = (quote: Quote | BoostQuote) => {
   }
 };
 
+export type NetworkStatus = inferRouterOutputs<AppRouter>['networkStatus'];
```

Comment:
think i would prefer making this type explicit. its always a pain to figure out library types if they are inferred somehow

Reply:
<img width="390" alt="image" src="https://github.com/user-attachments/assets/4ffd6a44-aa23-4841-b3c7-51c27bf9da90" />
this one is good, but also it's not intended for end users so i'll remove the export